go/runtime/registry/host_rofl_mgmt.go

```diff
@@ -87,6 +90,21 @@ func (rh *roflHostHandler) handleVolumeManagement(rq *enclaverpc.Request) (any,
 	}
 }
 
+// handleLogManagement handles log management local RPCs.
+func (rh *roflHostHandler) handleLogManagement(ctx context.Context, rq *enclaverpc.Request) (any, error) {
+	switch rq.Method {
+	case rofl.MethodLogGet:
+		// Get logs.
+		var args rofl.LogGetRequest
+		if err := cbor.Unmarshal(rq.Args, &args); err != nil {
+			return nil, err
+		}
+		return rh.handleLogGet(ctx, &args)
```

Comment:
How will runtime call this endpoint, as currently this method returns all lines, and not only new lines from the last call?

Reply:
Added a way to seek by timestamp.